veracruz-client/src/veracruz_client.rs

```diff
@@ -300,11 +279,33 @@ impl VeracruzClient {
         }
     }
 
-    pub fn get_results(&mut self, file_name: &str) -> Result<Vec<u8>, VeracruzClientError> {
+    pub fn request_compute(&mut self, path: &str) -> Result<Vec<u8>, VeracruzClientError> {
+        self.check_policy_hash()?;
+        self.check_runtime_hash()?;
+
+        let path = enforce_leading_backslash(path);
+        let serialized_read_result = transport_protocol::serialize_request_result(&path)?;
+        let response = self.send(&serialized_read_result)?;
+
+        let parsed_response = transport_protocol::parse_runtime_manager_response(&response)?;
+        let status = parsed_response.get_status();
+        if status != transport_protocol::ResponseStatus::SUCCESS {
+            return Err(VeracruzClientError::ResponseError("request_compute", status));
+        }
+        if !parsed_response.has_result() {
+            return Err(VeracruzClientError::VeracruzServerResponseNoResultError);
+        }
+        let response_data = &parsed_response.get_result().data;
+        return Ok(response_data.clone());
+    }
+
+
+    pub fn get_results(&mut self, path: &str) -> Result<Vec<u8>, VeracruzClientError> {
```

Comment:
Similar comments here regarding the type of `path` as above, and a lack of `rustdoc` comments on this public function.